File: WindowsFormsApplication2/WindowsFormsApplication2/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WindowsFormsApplication2
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            DialogResult resultado = MessageBox.Show("Mensaje", "Rotulo", MessageBoxButtons.YesNoCancel, MessageBoxIcon.Question);
            if (resultado == DialogResult.Yes)
                MessageBox.Show("Has seleccionado YES");
            if (resultado == DialogResult.No)
                MessageBox.Show("Has seleccionado NO");
            if (resultado == DialogResult.Cancel)
                MessageBox.Show("Has seleccionado CANCEL");
        }

        private void checkBox1_CheckedChanged(object sender, EventArgs e)
        {
            if(checkBox1.Checked==true)
                MessageBox.Show("JAJAJAJJA te la comiste");
        }

        
    }
}

File: WindowsApplication3/WindowsApplication3/Form1.vb
Public Class Form1

    Private Sub CB1_SelectedIndexChanged(sender As Object, e As EventArgs) Handles CB1.SelectedIndexChanged
            string a= CB1.SelectedItem.ToString();
            MessageBox.Show(a);
            T1.Text = a;
    End Sub

    Private Sub T1_TextChanged(sender As Object, e As EventArgs) Handles T1.TextChanged

    End Sub
End Class
